a11y/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM node
```

Comment:
This should really have a version specified. Also, you might consider adding or using a base image in https://github.com/ONSdigital/dp-concourse-tools

Reply:
Friday moment, just noticed this IS in dp-concourse-tools 🤦

Reply:
Good idea, otherwise this might randomly break on us as it would just use the latest. Have fixed it to a version now.